docs/PROJECT_OVERVIEW.md

```
@@ -16,6 +16,9 @@ O Vinheiro **não vende vinho**, não gerencia estoque físico e não faz entreg
 - [A Solução](#-a-solução)
 - [Funcionalidades](#-funcionalidades)
 - [Motor de Recomendação por IA](#-motor-de-recomendação-por-ia--funcionalidade-core)
+- [Modelo de Negócio](#-modelo-de-negócio)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)